pkg/oc/bootstrap/docker/up.go

```diff
@@ -390,6 +390,13 @@ func (c *ClusterUpConfig) Validate(errout io.Writer) error {
 	if c.dockerClient == nil {
 		return fmt.Errorf("missing dockerClient")
 	}
+	if c.command.Flag("enable").Changed {
```

Comment:
Naughty!  We shouldn't keep that command after `Complete`.  Just check if the value of `UserEnabledComponents` has changed from the default we choose.

Reply:
@deads2k well this is Validate() which is before Complete. What is wrong on relying on flag library telling us whether a flag was changed or not?

Reply:
OK, I was wrong... I always thought we do Validate() before we complete, in cluster up it is other way around...